transformation/src/jade-schema/tables/sequence_file.table.json

```diff
@@ -0,0 +1,159 @@
+{
+  "name": "sequence_file",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "byte_size",
+      "datatype": "integer",
+      "type": "required"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "data_modality",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "audit_label",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "audit_severity",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "award",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "file_format_type",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "lab",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "platform",
+      "datatype": "string"
+    },
+    {
+      "name": "quality_metrics",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "restriction_enzymes",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "status",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "submitted_by",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "genome_annotation",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "pipeline_id",
+      "datatype": "string"
+    },
+    {
+      "name": "step_run",
+      "datatype": "string"
```

Comment:
Do sequence files have a `step_run`?

Reply:
Good catch. We should only need the step_run field for alignment files.